src/Bicep.Core/Semantics/ParamsSemanticModel.cs

```diff
@@ -22,7 +22,11 @@ public class ParamsSemanticModel
         public ParamsTypeManager ParamsTypeManager { get; }
         public ParamsSymbolContext ParamsSymbolContext { get; }
         public Lazy<ImmutableArray<IDiagnostic>> AllDiagnostics { get; }
+        // public IFileResolver fileResolver { get; }
```

Comment:
Honestly not sure where `fileResolver` came from after taking a look at previous PRs, commented it out for now

Reply:
I think you can remove it if it's not used in this file.